vim_binding.js

```diff
@@ -119,6 +119,30 @@ define([
         }
       }
   };
+
+  // soft-failing shortcut helper - works around the hard-failing shortcut
+  // manager in IPython
+  var add_shortcuts = function add_shortcuts(manager_name, data){
```

Comment:
I would not like to mess global namespace so could you remove `add_shortcuts` part after `function`?